cli/config.go

```diff
@@ -24,10 +24,13 @@ func Config(ctx context.Context, logger *zap.Logger, cfg *config.Config, service
 		Short:   "manage keploy configuration file",
 		Example: "keploy config --generate --path /path/to/localdir",
 		PreRunE: func(cmd *cobra.Command, _ []string) error {
-			return cmdConfigurator.ValidateFlags(ctx, cmd)
+			if err := cmdConfigurator.ValidateFlags(ctx, cmd); err != nil {
+				utils.LogError(logger, err, "failed to validate flags")
+				return err
+			}
+			return nil
```

Comment:
what's the use of logerror?

Reply:
This is proper use of handling errors according to Google's Golang documentation. The purpose of it is to simply log errors following production standard.

Reply:
https://go.dev/blog/error-handling-and-go

Reply:
I appreciate the enthusiasm but the said logs aren't used. I put a print function  in utils.LogError and it doesn't work.

I understand its a official method of doings things but it can be cross checked if its the code snippet is working or not.



EXAMPLE: 
<img width="914" alt="image" src="https://github.com/keploy/keploy/assets/60891544/2a6c1dbf-dc06-48b8-879b-ad833e7bccf1">
****

Reply:
The LogError is a function that is used throughout the entirety of the codebase and was designed by the developers.

**What you're looking at is outside the scope of the issue you posted. The but has been fixed and is ready to be merged. **

Also to trigger that, you would need to make ValidateFlags fail.

If you have further questions about LogError() Ask the creators of Keploy themselves.

Reply:
Perhaps @slayerjain can provide some insight on the LogError function. But yes, the function the original developers implemented does work in the proper circumstances of error handling and detection.

Reply:
@ethanwater @shivamsouravjha its totally cool to use the `LogError` wrapper. It's something we recently added to avoid logging unnecessary errors while ctx is cancelled. We might add more validations in the future.

Reply:
Thank you @slayerjain for the review. I can implement these features. Should I implement these in the current PR or another one after merge?

Reply:
I would really appreciate implementing in the current PR. Happy to actively share my thoughts if you need :)

Reply:
@slayerjain Awesome, looking at it- I agree with these features. What do you think of this:

- ❌ and red text used for fatal errors/ return err
- 📌 or ⚠️ and yellow text used for info logs/warnings